falcon/app.py

```diff
@@ -159,6 +159,10 @@ def process_response(self, req, resp, resource, req_succeeded)
             (default ``False``).
             (See also: :ref:`CORS <cors>`)
 
+        sink_before_static_routes (bool): Indicates if the sinks should be processed
```

Comment:
Shouldn't we match the grammatical number of these terms inside the new parameter, i.e., `sinks_before_static_routes` or `sink_before_static_route`?